cmd/admin/pause_identifier_test.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"context"
+	"os"
+	"path"
+	"strings"
+	"testing"
+
+	blog "github.com/letsencrypt/boulder/log"
+	sapb "github.com/letsencrypt/boulder/sa/proto"
+	"github.com/letsencrypt/boulder/test"
+	"google.golang.org/grpc"
+)
+
+// mockSAPaused is a mock which records the PauseRequest it received, and
+// returns the number of identifiers as a PauseIdentifiersResponse. It does not
+// maintain state of repaused identifiers.
+type mockSAPaused struct {
+	sapb.StorageAuthorityClient
+	reqs []*sapb.PauseRequest
+}
+
+func (msa *mockSAPaused) PauseIdentifiers(ctx context.Context, in *sapb.PauseRequest, _ ...grpc.CallOption) (*sapb.PauseIdentifiersResponse, error) {
+	msa.reqs = append(msa.reqs, in)
+
+	return &sapb.PauseIdentifiersResponse{Paused: int64(len(in.Identifiers))}, nil
+}
+
+func TestPausingIdentifiers(t *testing.T) {
```

Comment:
nit: I'd split this test into two separate tests -- one for `readPausedAccountFile`, and one for `pauseIdentifiers`. That way each test can be a little bit more narrowly focused, with only the first test needing to write files to a temp dir, and only the second test needing a mock SA. It also allows for easier expansion in the future if we implement other input types.